### PR TITLE
Fixes for shell-cmds and remove inetutils

### PR DESCRIPTION
**Pull request by Gymxo**
Remove inetutils
Add pfetch ios support from upstream
Build hostname in shell-cmds

### CODE REVIEW

**Comment by Gymxo** [comment]
Don't accept pr until remote-cmds gets added*

**Comment by Diatrus** [comment]
Why remove inetutils

**Comment by Gymxo** [comment]
Added remote-cmds

File: build_info/remote-cmds.control [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Package: remote-cmds
+Version: @DEB_REMOTE-CMDS_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Depends: libedit0, libiosexec1, libncursesw6
```
Comment (CRKatri) [inline build_info/remote-cmds.control:5]: `Conflicts`, `Provides` for `inetutils`.

File: makefiles/remote-cmds.mk [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+		-ledit -lcurses -DUSE_TERMIO -framework CoreFoundation -framework IOKit; \
+	done
+ifneq (,$(findstring macosx,$(PLATFORM)))
+	mkdir -p $(BUILD_STAGE)/remote-cmds/$(MEMO_PREFIX){/Library/LaunchDaemons,/$(MEMO_SUB_PREFIX)/{bin,libexec,/share/man/man{1,8}}}
+	cp -a $(BUILD_WORK)/remote-cmds/telnet $(BUILD_STAGE)/remote-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin
+	cp -a $(BUILD_WORK)/remote-cmds/telnetd $(BUILD_STAGE)/remote-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/libexec
+	cp -a $(BUILD_WORK)/remote-cmds/telnetd.tproj/telnet.plist $(BUILD_STAGE)/remote-cmds/$(MEMO_PREFIX)/Library/LaunchDaemons
+	cp -a $(BUILD_WORK)/remote-cmds/telnet.tproj/*.1 $(BUILD_STAGE)/remote-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man1/
+	cp -a $(BUILD_WORK)/remote-cmds/telnetd.tproj/*.8 $(BUILD_STAGE)/remote-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man8/
+else ifneq (,$(findstring iphoneos,$(PLATFORM)))
```
Comment (CRKatri) [inline makefiles/remote-cmds.mk:48]:
```suggestion
else
```

File: makefiles/remote-cmds.mk [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	cd $(BUILD_WORK)/remote-cmds/libtelnet; \
+	$(CC) $(CFLAGS) -c *.c -D'__FBSDID=__RCSID' -I. -DHAS_CGETENT -DAUTHENTICATION -DRSA -DFORWARD -DHAVE_STDLIB_H; \
+	$(AR) -cr libtelnet.a *.o
+	cp -a $(BUILD_WORK)/remote-cmds/libtelnet/libtelnet.a $(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib
+	cp -a $(BUILD_WORK)/remote-cmds/wall.tproj/{ttymsg.c,ttymsg.h} $(BUILD_WORK)/remote-cmds/talkd.tproj
+	rm -rf $(BUILD_WORK)/remote-cmds/telnetd.tproj/{authenc.c,strlcpy.c}
+	cd $(BUILD_WORK)/remote-cmds && $(CC) $(CFLAGS) -o telnetd telnetd.tproj/*.c $(LDFLAGS) -lcurses -ltelnet -DENV_HACK -DUSE_TERMIO -DNO_UTMP -D'__FBSDID(x)='
+	cd $(BUILD_WORK)/remote-cmds && $(CC) $(CFLAGS) -o telnet telnet.tproj/*.c $(LDFLAGS) -ltelnet -lcurses -DUSE_TERMIO
+	cd $(BUILD_WORK)/remote-cmds; \
+	for tproj in logger talk talkd tftp tftpd wall; do \
```
Comment (CRKatri) [inline makefiles/remote-cmds.mk:36]: Don't build these for macOS.

File: makefiles/remote-cmds.mk [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+DEB_REMOTE-CMDS_V   ?= $(REMOTE-CMDS_VERSION)
+
+remote-cmds-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-$(REMOTE-CMDS_VERSION).tar.gz
+	wget -q -nc -P $(BUILD_SOURCE) https://opensource.apple.com/tarballs/libtelnet/libtelnet-$(LIBTELNET_VERSION).tar.gz
+	$(call EXTRACT_TAR,remote_cmds-$(REMOTE-CMDS_VERSION).tar.gz,remote_cmds-$(REMOTE-CMDS_VERSION),remote-cmds)
+	$(call EXTRACT_TAR,libtelnet-$(LIBTELNET_VERSION).tar.gz,libtelnet-$(LIBTELNET_VERSION),remote-cmds/libtelnet)
+	wget -q -nc -P $(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include https://opensource.apple.com/source/Libc/Libc-1044.1.2/include/sgtty.h
+	wget -q -nc -P $(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include https://opensource.apple.com/source/lukemftp/lukemftp-13/tnftp/libedit/histedit.h
+	wget -q -nc -P $(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/include/protocols https://opensource.apple.com/source/Libc/Libc-1044.1.2/include/protocols/talkd.h
```
Comment (CRKatri) [inline makefiles/remote-cmds.mk:19]: This belongs in `Makefile`

**Review by CRKatri — CHANGES_REQUESTED** [review]

**Comment by Gymxo** [comment]
Also added lukemftp (a ftp client)

**Comment by Gymxo** [comment]
this pr is a mess
also I forgot that toybox existed so

**Pull request closed without merging**